helm-chart-sources/webservice/templates/hook.yaml

```diff
@@ -6,11 +6,8 @@ kind: Job
 metadata:
   name: {{ $hookName }}
```

Comment:
Should this switch to generateName now, since we're not 'replacing' the previous hook?

Reply:
I haven't reverted us back to the "rely on TTL + unique job names" version. Unless specified the policy is BeforeHookCreation - I've just removed the ability to override this via the values. We are still replacing the previous hooks. I think we both said we don't care which we go last time. I still don't but this ticket wasn't about reverting that previous behaviour change.